Assets/Scripts/SpawnManager.cs

```diff
@@ -19,6 +19,8 @@ void Start()
 				SpawnPoints = GameObject.FindGameObjectsWithTag("Respawn");
 	            Vector3 spawnPositon = GetRandomSpawnPoint();
 	            LocalPlayer = PhotonNetwork.Instantiate("NetPlayer", spawnPositon, Quaternion.identity, 0);
+                // todo: Delete once event system is in place
+                LocalPlayer.GetComponent<LifeManager>().mDepositManager = GameObject.Find("DepositBox").GetComponent<DepositManager>();

```

Comment:
I don't. I'm setting the life manager's deposit manager using the scene's deposit box's Deposit Manager.

This is all about setting the Life Manager's deposit manager, not retrieving it